packages/compiler/templates/__snapshots__/library-ts/src/decorators.ts

```diff
@@ -0,0 +1,20 @@
+import { DecoratorContext, Operation, Program } from "@typespec/compiler";
+import { createStateSymbol, reportDiagnostic } from "./lib.js";
+
+export const namespace = "LibraryTs";
+
+const alternateNameKey = createStateSymbol("alternateName");
```

Comment:
Might be nice to put comments on the various parts here so it reads like a sample as well.

Reply:
I'll also change to the new way of doing state symbols